packages/opentelemetry-exporter-collector-proto/test/CollectorMetricExporter.test.ts

```diff
@@ -14,50 +14,44 @@
  * limitations under the License.
  */
 
+import * as api from '@opentelemetry/api';
+import { ExportResultCode } from '@opentelemetry/core';
 import {
-  collectorTypes,
   CollectorExporterNodeConfigBase,
+  collectorTypes,
 } from '@opentelemetry/exporter-collector';
-import * as api from '@opentelemetry/api';
 import * as metrics from '@opentelemetry/metrics';
-import * as http from 'http';
 import * as assert from 'assert';
+import * as http from 'http';
 import * as sinon from 'sinon';
 import { CollectorMetricExporter } from '../src';
 import { getExportRequestProto } from '../src/util';
-
 import {
-  mockCounter,
-  mockObserver,
-  ensureExportMetricsServiceRequestIsSet,
-  mockValueRecorder,
   ensureExportedCounterIsCorrect,
   ensureExportedObserverIsCorrect,
   ensureExportedValueRecorderIsCorrect,
+  ensureExportMetricsServiceRequestIsSet,
+  mockCounter,
   MockedResponse,
+  mockObserver,
+  mockValueRecorder,
 } from './helper';
-import { ExportResult, ExportResultCode } from '@opentelemetry/core';
-import { CollectorExporterError } from '@opentelemetry/exporter-collector/build/src/types';
 
 const fakeRequest = {
   end: function () {},
   on: function () {},
   write: function () {},
 };
 
-// send is lazy loading file so need to wait a bit
-const waitTimeMS = 20;
-
 describe('CollectorMetricExporter - node with proto over http', () => {
   let collectorExporter: CollectorMetricExporter;
   let collectorExporterConfig: CollectorExporterNodeConfigBase;
-  let spyRequest: sinon.SinonSpy;
-  let spyWrite: sinon.SinonSpy;
   let metrics: metrics.MetricRecord[];
+
+  const sandbox = sinon.createSandbox();
```

Comment:
Is the extra sandbox really needed here? Usually it's fine to use the default sandbox - the sinon import (e.g. use `sinon.stub` and in after `sinon.restore()`.

Reply:
No it would not be needed. Is there a drawback to using it?

Reply:
It's easier to do something wrong if there are two sandboxes.
Most people are used to just use `sinon.stub` which would result in not restoring here as the stub is in a different sandbox.
Usually a dedicated sanbox is only needed if you have the need to restore/reset a group of stubs/spys independent of the other.

Reply:
sounds good